doc/dev-guide/src/linting.md

```diff
@@ -14,24 +14,21 @@ $ cargo clippy --all --all-targets --all-features -- -D warnings
 
 ## Rust-Analyzer
 
-When using  [`rust-analyzer`](https://rust-analyzer.github.io/) integration in the IDE of your choice, you might want to set the `rust-analyzer.cargo.features` configuration to `"all"` (check the [`rust-analyzer` manual](https://rust-analyzer.github.io/manual.html#configuration) for more details).
+When using [`rust-analyzer`](https://rust-analyzer.github.io/)'s clippy integration in the IDE of your choice, apart from setting the `rust-analyzer.check.command` property to `"clippy"`, you might also want to set the `rust-analyzer.cargo.features` property to `"all"` (check the [`rust-analyzer` manual](https://rust-analyzer.github.io/manual.html#configuration) for more details).
```

Comment:
Pre-existing, but if you're goint to improve this:

* Setting the `check.command` is entirely independent of setting `features` to `all`, since the latter is useful even if you don't change the `command`.
* Would use a separate sentence for the pointer to Rust-Analyzer docs instead of wrapping it in parentheses.